src/models/propulsion/FGBldc.h

```diff
@@ -0,0 +1,100 @@
+/*%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+ Header:       FGBldc.h
+ Author:       Matt Vacanti
+ Date started: 11/08/2020
+
+ ----- Copyright (C) 2020  Matt Vacanti (mdvacanti@gmail.com) --------------
+
+UPDATE LICENSE
```

Comment:
```suggestion
 --------- Copyright (C) 2020-2022  Matt Vacanti / Paolo Becchi-------------

 This program is free software; you can redistribute it and/or modify it under
 the terms of the GNU Lesser General Public License as published by the Free
 Software Foundation; either version 2 of the License, or (at your option) any
 later version.

 This program is distributed in the hope that it will be useful, but WITHOUT
 ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
 FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more
 details.

 You should have received a copy of the GNU Lesser General Public License along
 with this program; if not, write to the Free Software Foundation, Inc., 59
 Temple Place - Suite 330, Boston, MA 02111-1307, USA.

 Further information about the GNU Lesser General Public License can also be
 found on the world wide web at http://www.gnu.org.
```
The license shall be LGPL 2.0+. Note that you'll need an agreement from @mvacanti for that.